scenes/game/local_game.gd

```diff
@@ -3518,16 +3518,37 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			decision_info.choice = [{
 				"effect_type": "pass"
 			}]
-			decision_info.limitation = [0]
+
+			var move_min = 0
+			var move_max = 9
+			if 'move_min' in effect:
+				move_min = effect['move_min']
+			if 'move_max' in effect:
+				move_max = effect['move_max']
+
+			decision_info.limitation = []
+			if move_min == 0:
+				decision_info.limitation.append(0)
+				decision_info.choice.append({
+					"effect_type": "move_to_space",
```

Comment:
this also needs to be added if say you're at location 9 and move_min and move_max > 1, to account for retreat 1 into corner.

Reply:
also, we should double check that this works with required movement, move_min>1 and movement is blocked.

Reply:
should be improved after the latest push; I held off on applying it to existing movement effects but I can try to if it looks good

Reply:
Let's do applying it to other effects as a separate change.